src/views/requestAccess.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
+import { useHistory } from 'react-router-dom';
+import { ACCESS_REQUEST_DATA } from '../components/DashboardModule/data';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
+
+export const Footer = () => {
```

Comment:
We should move this footer to App.js since it should be on nearly every page instead of just the Dashboard or RequestAccess pages